### ISSUE TITLE
Filter mutaion/queries logging with a custom function

### ISSUE THREAD

**Issue by kandros** [blitz-js/blitz#2027]
### What do you want and why?

Ability to filter which logs will be printed in the console.

There is a lot of noise generated by queries such as `getCurrentUser`

### Possible implementation(s)

a function declared inside `blitz.config.js` that receives the message going to be logged and extra metadata when available.

### Additional context
I have thousands of logs like that that are not much useful, I would like to be able to disable them a per case basis

![image](https://user-images.githubusercontent.com/4562878/109499241-e7f0d280-7a94-11eb-93c3-6d5221a88abf.png)

**Comment by flybayer** [comment]
Great suggestion!

**Comment by kandros** [comment]
Wondering if is better to make `rpcMiddleware` https://github.com/blitz-js/blitz/blob/5619dcca97317e61cc1948493c40d3a578c01c1a/packages/server/src/rpc.ts#L43 somehow configurable (maybe via exporting a config object) or intercept logging at the `display` package level https://github.com/blitz-js/blitz/blob/canary/packages/display/src/index.ts

**Comment by flybayer** [comment]
@kandros yeah the config idea is pretty good. What do you picture it looking like?

**Comment by kandros** [comment]
Something like 

```
// app/queries/getCurrentUser.tsx

export const rpcMiddlewareConfig = { loggingEnabled: false }

export default async function getCurrentUser(args: Args, ctx: Ctx) {
  ...
}
```

Where in the code can I find the point where `getCurrentUser` gets imported?

**Comment by flybayer** [comment]
Ok, let's make it a bit more general, so like this:

```
export const config = {
  log: false
}
```

And the current rpc code is getting ready to be totally changed with the next.js fork, so best to wait on implementing this.

A workaround is to use https://github.com/ds300/patch-package to patch the display package.

**Comment by Roesh** [comment]
Hey, wondering if we have updated the rpc code since nextjs has been forked? I'd be down to help with this too since I'm doing a query with refetch interval of 500ms and its flooding my logs haha

**Comment by kandros** [comment]
> Hey, wondering if we have updated the rpc code since nextjs has been forked? I'd be down to help with this too since I'm doing a query with refetch interval of 500ms and its flooding my logs haha

If it has and can be worked on, I'm available for doing it in pair programming you are open to it.
